dap_aria_mapping/getters/taxonomies.py

```diff
@@ -64,7 +64,7 @@ def get_cooccurrence_taxonomy() -> pd.DataFrame:
     """
     return download_obj(
         BUCKET_NAME,
-        "outputs/community_detection_clusters.parquet",
+        "outputs/community_detection_taxonomy/tax.parquet",
```

Comment:
This is what I meant with the naming conflicts, for consistency this should be `cooccur_taxonomy`. But not too important.

Reply:
yeah - I think we should open a new issue for this and change it once things have been merged to dev otherwise there will be a lot of merging messiness as I have many open PRs right now that aren't consistent